Silicon/Intel/Tools/FitGen/FitGen.c

```diff
@@ -3578,6 +3627,11 @@ GetFitEntryInfo (
       gFitTableContext.ProtBootPolicy.Version = FitEntry[FitIndex].Version;
       gFitTableContext.ProtBootPolicy.Size = GetFirmwareInterfaceTableEntrySize (&FitEntry[FitIndex]);
       break;
+    case FIT_TABLE_TYPE_MMC_FW:
+      gFitTableContext.MmcFw[gFitTableContext.MmcFwNumber].Address = (UINT32)FitEntry[FitIndex].Address;
+      gFitTableContext.MmcFw[gFitTableContext.MmcFwNumber].Version = FitEntry[FitIndex].Version;

```

Comment:
don't we need to update the size as part of FIT ? Looks like it is missing.

Reply:
You mean that "gFitTableContext.MmcFw[gFitTableContext.MmcFwNumber].Size", right?
We have update this variable on GetFitEntryNumber(). So, I think that we don't update this again.